requirements/django.txt

```diff
@@ -1 +1 @@
-django==1.11.22
+django==1.11.24
```

Comment:
Looks like 1.11.25 is now available. Want to update to that version? If so, lgtm.

Reply:
Yup.